backend/src/test/java/mouda/backend/moim/domain/MoimTest.java

```diff
@@ -0,0 +1,237 @@
+	private static final String PLACE = "서울시 동작구 강원대로 10길 5";
+	private static final int MAX_PEOPLE = 11;
+	private static final String AUTHOR_NICKNAME = "안나";
+	private static final String DESCRIPTION = "이번 주 금요일에 퇴근하고 축구하실 분 계신가요? 끝나고 치맥도 할 생각입니다.";
+
+	@DisplayName("모임 객체를 정상적으로 생성한다.")
+	@Test
+	void createMoim() {
+		Assertions.assertDoesNotThrow(() -> Moim.builder()
+			.title(TITLE)
+			.date(DATE)
+			.time(TIME)
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("제목 길이가 제한을 초과하면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenTitleIsTooLong() {
+		String longTitle = "a".repeat(31);
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title(longTitle)
+			.date(DATE)
+			.time(TIME)
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("제목이 빈 문자열이면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenTitleDoesNotExists() {
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title("")
+			.date(DATE)
+			.time(TIME)
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("날짜가 null이면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenDateIsNull() {
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title(TITLE)
+			.date(null)
+			.time(TIME)
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("시간이 null이면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenTimeIsNull() {
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title(TITLE)
+			.date(DATE)
+			.time(null)
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("모임 날짜가 현재보다 과거이면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenDateIsPast() {
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title(TITLE)
+			.date(LocalDate.now().minusDays(1))
+			.time(LocalTime.now().plusHours(1))
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("모임 시간이 현재보다 과거이면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenTimeIsPast() {
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title(TITLE)
+			.date(LocalDate.now())
+			.time(LocalTime.now().minusHours(1))
+			.place(PLACE)
+			.maxPeople(MAX_PEOPLE)
+			.authorNickname(AUTHOR_NICKNAME)
+			.description(DESCRIPTION)
+			.build());
+	}
+
+	@DisplayName("장소가 빈 문자열이면 모임 객체 생성에 실패한다.")
+	@Test
+	void failToCreateMoimWhenPlaceIsBlank() {
+		assertThrows(MoimException.class, () -> Moim.builder()
+			.title(TITLE)
+			.date(LocalDate.now())
+			.time(LocalTime.now().minusHours(1))
```

Comment:
```suggestion
			.date(DATE)
			.time(TIME)
```
과거 시간이어서 예외가 발생할 수 있을 것 같아요~!

Reply:
과거 시간을 테스트하는 게 맞아요 😀